incubator/exporter-prometheus.spec/src/main/scripts/io/aklivity/zilla/specs/exporter/prometheus/schema/prometheus.schema.patch.json

```diff
@@ -0,0 +1,50 @@
+[
+    {
+        "op": "add",
+        "path": "/$defs/telemetry/exporter/oneOf/-",
+        "value":
+        {
+            "properties":
+            {
+                "type":
+                {
+                    "const": "prometheus"
+                },
+                "options":
+                {
+                    "properties":
+                    {
+                        "endpoints":
+                        {
+                            "type": "array",
+                            "items":
+                            {
+                                "properties":
+                                {
+                                    "scheme":
+                                    {
+                                        "title": "Scheme",
+                                        "type": "string"
+                                    },
+                                    "port":
+                                    {
+                                        "title": "Port",
+                                        "type": "int"
+                                    },
+                                    "path":
+                                    {
+                                        "title": "Path",
+                                        "type": "string"
+                                    }
+                                },
```

Comment:
Please mark `scheme` and `port` as required.

Reply:
done